addons/hw_escpos/escpos/constants.py

```diff
@@ -75,7 +75,6 @@
 TXT_ENC_PC862   = '\x1b\x74\x24' # PC862 Hebrew
 TXT_ENC_PC864   = '\x1b\x74\x25' # PC864 Arabic
 TXT_ENC_PC869   = '\x1b\x74\x26' # PC869 Greek
-TXT_ENC_PC936   = '\x1C\x21\x00' # PC936 GBK(Guobiao Kuozhan)
```

Comment:
Should not be removed: This was added last month by 8fe042ea917dad6a09fe69ef47ad6d6c4f3a72af